dyninstAPI/src/inst-aarch64.C

```diff
@@ -1424,13 +1424,139 @@ bool EmitterAARCH64Dyn::emitTOCCommon(block_instance *block, bool call, codeGen
 
 // TODO 32/64-bit?
 bool EmitterAARCH64Stat::emitPLTCall(func_instance *callee, codeGen &gen) {
-    assert(0); //Not implemented
-    return emitPLTCommon(callee, true, gen);
+    /*
```

Comment:
If the commented-out code is acting as documentation, please add more comments to explain. If it's just dead code, please remove it.

Reply:
It's praxis to keep old cold commented out or disabled in case you find a bug in a the new implementation or you need to revert something. Searching back to git history is not as practical, and even worse when you are using non-git functionality that are dependent on a single vendor and not supported by git compliant clients, such as PRs.